adapters/dynamoDB_adapters/src/main/java/org/atomhopper/dynamodb/adapter/DynamoFeedDBPublisher.java

```diff
@@ -0,0 +1,218 @@
+package org.atomhopper.dynamodb.adapter;
+
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDBClient;
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBMapper;
+import com.amazonaws.services.dynamodbv2.document.*;
+import com.amazonaws.services.dynamodbv2.document.spec.QuerySpec;
+import com.amazonaws.services.dynamodbv2.document.utils.ValueMap;
+import com.yammer.metrics.Metrics;
+import org.apache.abdera.model.Entry;
+import org.apache.commons.lang.StringUtils;
+import org.atomhopper.adapter.FeedPublisher;
+import org.atomhopper.adapter.NotImplemented;
+import org.atomhopper.adapter.PublicationException;
+import org.atomhopper.adapter.ResponseBuilder;
+import org.atomhopper.adapter.request.adapter.DeleteEntryRequest;
+import org.atomhopper.adapter.request.adapter.PostEntryRequest;
+import org.atomhopper.adapter.request.adapter.PutEntryRequest;
+import org.atomhopper.dynamodb.constant.DynamoDBConstant;
+import org.atomhopper.dynamodb.model.PersistedEntry;
+import org.atomhopper.response.AdapterResponse;
+import org.atomhopper.response.EmptyBody;
+import org.atomhopper.util.uri.template.EnumKeyedTemplateParameters;
+import org.atomhopper.util.uri.template.URITemplate;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import com.yammer.metrics.core.Counter;
+
+import java.io.IOException;
+import java.io.StringWriter;
+import java.text.Format;
+import java.text.SimpleDateFormat;
+import java.util.*;
+
+import static org.apache.abdera.i18n.text.UrlEncoding.decode;
+
+/**
+ * @Author: shub6691
+ * This class is used to publish the records in dynamodb using EntryId,feedName .
+ * Index has been created on the feedName for fast search and getting the entry from the DynamoDb.
+ */
+public class DynamoFeedDBPublisher implements FeedPublisher {
```

Comment:
Rename the file to DynamoDBFeedPublisher

Reply:
Done